composer/tracing/src/TraceLogs.tsx

```diff
@@ -19,30 +19,54 @@
 import * as React from "react";
 import ToolBar from './ToolBar';
 import TraceList from './TraceList';
-import './index.scss';
+// import './index.scss';
 
 export interface TraceLogsState {
     filteredTraces: Array<any>;
+    selected?: string;
 }
 
 export interface TraceLogsProps {
     filters: any;
     clearLogs: Function;
     traces: Array<any>;
-    selected: string;
+    onSelected: Function;
 }
 
 export default class TraceLogs extends React.Component<TraceLogsProps, TraceLogsState> {
     static defaultProps = {
         traces : [],
+        onSelected: ()=>{},
     }
     constructor(props: any, context: any) {
         super(props);
         this.onFilteredTraces = this.onFilteredTraces.bind(this);
+        this.onSelected = this.onSelected.bind(this);
         this.state = {
             filteredTraces: props.traces,
         };
     }
+
+    onSelected(selected: any) {
+        if (selected.id === this.state.selected) {
+            this.setState({
+                selected: undefined
+            });
+            this.props.onSelected(undefined);
```

Comment:
Can we use boolean value instead `undefined` ?

```suggestion
            this.props.onSelected(false);
```

Reply:
selected is a `string` here